embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/delivery/EmbraceCacheService.kt

```diff
@@ -160,6 +160,20 @@
         }
     }
 
+    @Suppress("UNCHECKED_CAST")
+    override fun loadOldPendingApiCalls(name: String): List<PendingApiCall>? {
+        val file = File(storageDir, EMBRACE_PREFIX + name)
+        try {
+            val results = serializer.fromJson(file.inputStream(), ArrayList::class.java)
+            return results as List<PendingApiCall>? ?: return null
```

Comment:
Is there utility to return an empty list here so we can differentiate between the failure and nothing-found cases? If not, this is fine

Reply:
Updated to return an empty list.